cppcache/integration/framework/Cluster.cpp

```diff
@@ -393,6 +395,11 @@ void Cluster::applyLocators(apache::geode::client::PoolFactory &poolFactory) {
   }
 }
 
+void Cluster::applyServer(apache::geode::client::PoolFactory &poolFactory, 
```

Comment:
is `apply` the right verb for this function?  This function is actually adding a server (based on `poolFactory.addServer` call)

Reply:
On the same way locator is added.
`
void Cluster::applyLocators(apache::geode::client::PoolFactory &poolFactory) {`